package.json

```diff
@@ -1,8 +1,18 @@
 {
   "name": "@yext/answers-headless-react",
   "version": "1.2.0",
+  "description": "Answers Headless React is the official React UI Bindings layer for Answers Headless",
   "main": "./lib/esm/src/index.js",
   "types": "./lib/esm/src/index.d.ts",
+  "keywords": [
+    "search",
+    "nlp",
+    "ai",
+    "answers",
+    "yext",
+    "react",
+    "components"
```

Comment:
did product want "components" as a keyword here even though this library doesn't provide any components?

Reply:
This library technically has a component (AnswersHeadlessProvider) though it's really not about components

Reply:
Product only gave me one list, so I updated the repos with the same list. Should I ask product to give a list for every public repository?

Reply:
Yeah, I would ask for a list for each repo since they have different purposes